Views/backend/connect/view/import/local_products.js

```diff
@@ -83,9 +85,18 @@ Ext.define('Shopware.apps.Connect.view.import.LocalProducts', {
                     return '<span style="display:block; margin: 0 auto; height:25px; width:25px;" class="' + checked + '"></span>';
                 }
             }, {
-                header: 'Preis (brutto)',
+                header: 'HEK',
                 xtype: 'numbercolumn',
                 dataIndex: 'Detail_purchasePrice',
+                flex: 2,
+                renderer: function (value, meta) {
+                    meta.tdAttr = 'data-qtip="Händlereinkaufspreis"';
```

Comment:
I'm wondering only for the snippets. In most cases we use snippets instead of hard coded string.